packages/core/src/components/forms/_file-upload.scss

```diff
@@ -16,12 +16,16 @@ Markup:
 </label>
 
 :disabled - Disabled
+.pt-large - Larger size
 .pt-fill - Take up full width of parent element
 
 Styleguide pt-file-upload
 */
 
+$file-upload-input-selector: ".pt-file-upload-input";
```

Comment:
this is not a pattern we follow elsewhere... just repeat the selector instead of pulling it out to a string.